apartment/views.py

```diff
@@ -1,3 +1,45 @@
-from django.shortcuts import render
+import json
+from itertools import chain
 
-# Create your views here.
\ No newline at end of file
+from django.views import View
+from django.http import JsonResponse
+
+from apartment.models import ApartmentComplex, Size, District, Neighborhood
+from facility.models import School, Subway
+
+class SearchView(View):
+    def get(self, request):
+        try:
+            search = request.GET.get('search')
+
+            complexes     = ApartmentComplex.objects.filter(name__icontains=search)
+            districts     = District.objects.filter(name__icontains=search)
+            neighborhoods = Neighborhood.objects.filter(name__icontains=search)
+            schools       = School.objects.filter(name__icontains=search)
+            subways       = Subway.objects.filter(name__icontains=search)
+            total         = list(chain(districts, neighborhoods, schools, subways))
+
+            complex_lists = [{
+                'name'      : search.name,
+                'latitude'  : search.latitude,
+                'longitude' : search.longitude,
+                'address'   : search.address
+            }for search in complexes]
+
+            else_lists = [{
+                'name'      : search.name,
+                'latitude'  : search.latitude,
+                'longitude' : search.longitude
+            }for search in total]
+
+            lists = complex_lists + else_lists
+
+            if not lists:
+                return JsonResponse({"message":"NO_RESULT"}, status=400)
```

Comment:
```suggestion
                return JsonResponse({"message":"NO_RESULT"}, status=400)
```
위 메세지는 약속된건가요?
근데 사실 검색에서는 별다른 검색결과가 없다면 빈리스트를 반환해주는게 맞습니다.

Reply:
네! 저도 빈값이 맞나 에러처리 하는게 맞나 고민했었는데 감사합니다.